gripper_controller:
    type: effort_controllers/JointPositionController
    joint: claw_joint
    pid : {p: 10, i: 0.1, d: 0.1}

/gazebo_ros_control:
    pid_gains:
      claw_joint: {p: 10, i: 0.1, d: 0.1}